src/Agent/NewRelic/Agent/Core/DataTransport/ConnectionHandler.cs

```diff
@@ -88,6 +88,8 @@ public void Connect()
 
                     var securityPoliciesConfiguration = new SecurityPoliciesConfiguration(preconnectResult.SecurityPolicies);
                     EventBus<SecurityPoliciesConfigurationUpdatedEvent>.Publish(new SecurityPoliciesConfigurationUpdatedEvent(securityPoliciesConfiguration));
+
+                    _agentHealthReporter.ReportSupportabilityCountMetric(MetricNames.GetSupportabilityFeatureEnabled("LASP"));
```

Comment:
We may need to change this to `CSP`, because if I'm remembering things correctly, the original documentation and announcement referred to the feature as `CSP`, and `LASP` was the name used during the original development.

Reply:
According to the spec it's still LASP internally and CSP externally. Since this is a Supportability metric we may want the internal name but either is probably fine.

Reply:
I called it out because customers can see supportability metrics. An alternative approach is to consider if we can extract this information in BMDS instead.

Reply:
I'll change it to CSP instead. Easy peasy.